### DISCUSSION TITLE
New Team Members

### DISCUSSION THREAD

**Discussion by jankapunkt** [node-oauth/node-oauth2-server#268] [General]
Hey Team @HappyZombies @jorenvandeweyer we could need some more people on the team and I found that @shrihari-prakash and also more recently @dhensby showed valuable commitment here.

I see @shrihari-prakash has experiences with TypeScript, uses the lib in live setup for a while and has contributed a lot with helping feedback during the major transition towards 5.0.0.

@dhensby shows a well understanding of the standard and a massive experience with open source and software engineering in general, as I excerpted from your profile.

I would propose  to add you both to the org, to give you more freedom of contribution, but also to have you being part of future design decisions and being a helping hand in PR reviews.

So my question is to both sides: are you okay with being added to the org and shape the future of this lib and are the current team members okay with this, too?

In my opinion this would be a terrific move and would help in making this lib a viable option on he market.

Note: if you don't want to commit to this project that deep then this is totally fine and we can continue to collaborate the way we just did before.

**Comment by shrihari-prakash** [comment]
Hello @jankapunkt , that is okay from my side. I would be available to contribute to shape the lib. 💯

**Comment by jorenvandeweyer** [comment]
I totally agree, I think it would be a great addition to the team!

**Comment by dhensby** [comment]
I'd be more than happy to help out with the lib and provide guidance and input to help going forward 😊 

Thanks for the suggestion
